apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -70,12 +72,29 @@ li.ui-state-highlight {
   margin-bottom: 5px;
 }
 
+.pl-block-horizontal > .list-group {
```

Comment:
Why do these class names use a `pl-block-` prefix instead of the usual `pl-order-blocks-`?